version-control/tutorial/README.md

```diff
@@ -0,0 +1,13 @@
+# Super Sandwich Survey
+
+Sandwiches are a staple part of many people’s diets, particularly in western culture. However, little data exists on sandwich eating habits today. This repository will be used as a reference guide for anyone taking part in the super sandwich survey. 
+
+Thank you for taking part in the super sandwich survey. Your responses will contribute to a larger resource on people’s experiences with sandwiches. Please read the instructions file to learn more information about how to interact with the sandwich survey and how to log a new sandwich that is not currently in the sandwich database.
+
```

Comment:
I think it'd be good here to mention what files are contained in the folder and what they're for.